internal/frr/config.go

```diff
@@ -68,6 +68,7 @@ type NeighborConfig struct {
 	ASN             string
 	SrcAddr         string
 	Addr            string
```

Comment:
not sure I like that `Addr` can now be both interface or address, but for now can you add a comment about it please?

Reply:
comment added, but we can sure discuss what implementation you would like. Any suggestions?